symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/signal/OboSignalService.java

```diff
@@ -0,0 +1,128 @@
+package com.symphony.bdk.core.service.signal;
+
+import com.symphony.bdk.gen.api.model.BaseSignal;
+import com.symphony.bdk.gen.api.model.ChannelSubscriber;
+import com.symphony.bdk.gen.api.model.ChannelSubscriptionResponse;
+import com.symphony.bdk.gen.api.model.Signal;
+
+import org.apiguardian.api.API;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Service interface exposing OBO-enabled endpoints to manage signals information.
+ */
+@API(status = API.Status.STABLE)
+public interface OboSignalService {
+
+  /**
+   * Lists signals on behalf of the user.
+   * {@link SignalService#listSignals(Integer, Integer)}
+   *
+   * @param skip  The number of signals to skip.
+   * @param limit Maximum number of signals to return. Default is 50, maximum value is 500.
+   * @return List of signals that the user has created and public signals to which they have subscribed.
+   * @see <a href="https://developers.symphony.com/restapi/reference#list-signals">List Signals</a>
+   */
+  List<Signal> listSignals(Integer skip, Integer limit);
+
+  /**
+   * Lists signals on behalf of the user.
+   * {@link SignalService#listSignalsStream(Integer, Integer)}
+   *
+   * @param chunkSize size of elements to retrieve in one call. Optional and defaults to 50.
+   * @param totalSize size of elements to retrieve in one call. Optional and defaults to 50.
```

Comment:
Same description for those 2 params? Also, why not `skip` and `limit`? Looks confusing

Reply:
I took inspiration from this ticket https://perzoinc.atlassian.net/browse/APP-3183. I don't know why but we want to provide both of default pagination strategy (skip and limit) and our own pagination (chunksize and totalsize).